tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -16,9 +16,8 @@ class ViewImage(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == 'Gallery')
+        self.wait_to_be_displayed()
```

Comment:
We're already in the constructor of Gallery. Doing `Gallery(self.marionette).wait_to_be_displayed()` will provoke a endless loop. `self.wait_to_be_displayed()` is absolutely fine.